doc/datomic_differences.md

```diff
@@ -12,7 +12,7 @@ Without differences in the signature you may use the following like in Datomic:
 - datoms
 - [db](#db)
 - history
-- [index-range](#index-range)
+- [index-range](#index-range) -> dead link
```

Comment:
should be removed since we don't have this function

Reply:
Done